src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexPluginExtension.java

```diff
@@ -44,6 +44,10 @@
 
   private boolean allConfigurations;
 
+  private Set<String> modulesIncluded;
+
+  private Set<String> modulesExcluded;
```

Comment:
Could you please add this 2 properties at [OssIndexAuditTask.java](https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/fd1746863681929050bad0dc04d1b63995bd0954/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java#L248-L250) with the `@Input` annotation at the bottom?

Reply:
Sure, thanks for pointing that out.